WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -45,19 +47,31 @@ public  async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo i
 			Suggestions.Add(new PocketSuggestionViewModel(SmartLabel.Merge(transaction.SpentCoins.Select(x => CoinHelpers.GetLabels(x)))));
 		}
 
-		var suggestions =
-			await ChangeAvoidanceSuggestionViewModel.GenerateSuggestionsAsync(info, destination, wallet, transaction);
+		var loadingRing = new LoadingSuggestionViewModel();
+		Suggestions.Add(loadingRing);
 
 		var hasChange = transaction.InnerWalletOutputs.Any(x => x.ScriptPubKey != destination.ScriptPubKey);
 
+		IsLoading = false;
```

Comment:
What is the effect of this? Do we still need this, besides the loadingRing?

Reply:
It's obsoleted, we don't need it anymore IMO.